src/commands/Tools/create-emoji.ts

```diff
@@ -0,0 +1,48 @@
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { SkyraCommand } from '#lib/structures';
+import { PermissionLevels } from '#lib/types/Enums';
+import { ApplyOptions } from '@sapphire/decorators';
+import { EmojiRegex } from '@sapphire/discord.js-utilities';
+import { Args } from '@sapphire/framework';
+import { Message } from 'discord.js';
+
+@ApplyOptions<SkyraCommand.Options>({
+	aliases: ['add-emoji'],
+	cooldown: 10,
+	runIn: ['text'],
+	permissionLevel: PermissionLevels.Moderator,
+	permissions: ['MANAGE_EMOJIS'],
+	description: LanguageKeys.Commands.Tools.CreateEmojiDescription,
+	extendedHelp: LanguageKeys.Commands.Tools.CreateEmojiExtended
+})
+export class UserCommand extends SkyraCommand {
+	public async run(message: Message, args: SkyraCommand.Args) {
+		const emojiData = await args.pick(UserCommand.emojiResolver);
+
+		if (message.guild!.emojis.cache.has(emojiData.id))
```

Comment:
Once the above suggestion is done, do this:
```suggestion
		if (message.guild.emojis.cache.has(emojiData.id))
```

Reply:
Also, did you check `emojis.create` doesn't throws if there's an emoji named the same?